sdcm/tester.py

```diff
@@ -169,7 +169,12 @@ def setUp(self):
         logging.getLogger('botocore').setLevel(logging.CRITICAL)
         logging.getLogger('boto3').setLevel(logging.CRITICAL)
         self.init_resources()
-        self.db_cluster.wait_for_init()
+        if self.params.get('seeds_first', default='false') == 'true':
```

Comment:
Better way is to use one variable called for example  `first_seeds_num`.
The logic should be:
If `first_seeds_num` > 1 then
- self.db_cluster.wait_for_init(node_list=self.db_cluster.nodes[:seeds_num]) 
- self.db_cluster.wait_for_init(node_list=self.db_cluster.nodes[seeds_num:])

else:

- self.db_cluster.wait_for_init()

In addition `first_seeds_num` should be added to all YAMLs otherwise we will not be aware of its existence.

Reply:
We don't need the "seeds_first" param.
If the correct behaviour is to start the seeds first, we should always start the seeds_first.

So eventually we should have only 1 param the is visible to the user in the YAML and it's "seeds_num".
Then, the logic will be:
```
if seeds_num > 1:
    bla bla
else:
    bla
```

Reply:
?